.buildkite/longruns/pipeline.yml

```diff
@@ -1,22 +1,19 @@
+agents:
+  queue: central
+  slurm_mem: 8G
```

Comment:
Ahh okay, thank you

Reply:
Do you think it may be similarly problematic to specify both `slurm_ntasks` and `slurm_tasks_per_node`? e.g. [here](https://github.com/CliMA/ClimaCoupler.jl/blob/main/.buildkite/longruns/pipeline.yml#L100-L102)

Reply:
Yes, I think that can do weird things.